scripts/mkcloud

```diff
@@ -526,7 +543,7 @@ function setupadmin()
     fi
     echo "you can now proceed with installing crowbar"
     # prevent jumbo frames from going out
-    if [[ $want_mtu_size ]] ; then
+    if [[ $want_mtu_size -gt 1500 ]] ; then
```

Comment:
Possibly but not at this form. What could fix the problem is to use something like

```
if [[ $want_mtu_size -gt $host_mtu ]] ; then
....
fix the mss to be $host_mtu in iptables as well
```

I am going to give this solution a shot as well.